File: FrontEndv3/vueServer/src/Interfaces/CustodialAddressingServices/IServiceSettings.ts
import {INetworkSettings} from './INetworkSettings';
import {IDatabaseSettings} from './IDatabaseSettings';
import {ICasSettings} from './ICasSettings'; 
export interface IServiceSettings {
    networkSettings: INetworkSettings;
    databaseSettings: IDatabaseSettings;
    casSettings: ICasSettings;
  }

File: Custodial.BoilerPlate.Core/Custodial.BoilerPlate.Core/Interfaces/IServiceSettings.cs
using System;
using System.Collections.Generic;
using System.Reflection;
using System.Text;
using System.Threading.Tasks;

namespace Custodial.BoilerPlate.Core.Interfaces
{
    public interface IServiceSettings
    {
        INetworkSettings networkSettings { get; }
        IDatabaseSettings databaseSettings { get; }
        ICustodialAddressingSettings casSettings { get; }
        Task InitServiceSettingsAsync(string resourceFile, Assembly assembly);
    }
}
